index.html

```diff
@@ -6,51 +6,71 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.10.2/css/all.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ==" crossorigin="anonymous" referrerpolicy="no-referrer"/>
     <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
-    </head>
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+</head>
+
 <body>
+<!-- The header part of the code begins here -->
     <header>
-    <div class="pure-nav-list">
-            <h4 class="important"><a href="#">Ayodele</a></h4>
-            <nav>
-                <div class="menu1">
-                    <div class="menu"></div>
-                    <div class="menu"></div>
-                    <div class="menu"></div>
-                </div>
+        <div class="flexcontainer-header">
+            <a href="#"> Ayodele </a>
+            <nav class="navigation">
+                <ul>
+                <li><i class="fas fa-bars"></i></li>
+                </ul>
             </nav>
-            </div>
-            
-        </header>
+        </div>
+    </header>
+    <!-- The header part of the code ends here -->
 
-        <main>
-    <section class="section-headline">
-    <div class="main-text Developer">
-    <div class="introduction">
-    <h1>Hey There. <br> I’m Ayodele</h1>
-    <h3>I’am a Software Developer</h3>
-    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-    
+<!-- The content part of the code begins here -->
+<main class="flexcontainer-content">
+    <section class="box-content headline">
+    <h1>Hey There. <br> I'm Ayodele</h1> 
+    <div class="box-content product">
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
     </div>
+    <!-- <p class="box-content skill">I'm a software Developer!</p>
+        <P class="box-content product">I can help you build a product, feature or <br> website. Look through some of
+        my work <br> and Experience! if you like what you see <br> and have a project you need coded, <br> dont
+        hesitate to contact me.
+    </p> -->
```

Comment:
- [ ] Please try to keep your code clean and remove unnecessary comments. It's ok to put comments as long as they explain complex parts of your code or what functionality they produce. but please remember to always remove unnecessary commented code 👍.